.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
@KasperAndersson Let's use 16.15.1 (latest LTS) here, then regenerate the lock files.